torch/csrc/jit/python_ir.cpp

```diff
@@ -678,6 +678,12 @@ void initPythonIRBindings(PyObject* module_) {
         }
         return types;
       });
+  py::class_<TupleType::NamedTupleSpec>(m, "_NamedTupleSpec")
```

Comment:
This appears unused. Let's not bind something into python if it isn't used there. It is hard to prevent users from using it if it is there.